openflow_input/standard-1.4

```diff
@@ -2671,7 +2793,5 @@ struct of_requestforward : of_header {
     uint8_t type == 32;
     uint16_t length;
     uint32_t xid;
-    uint32_t role;
-    // TODO support embedding of_header
-    of_octets_t data;
+    of_header_t request;
```

Comment:
The build of the C-library currently fails, because of_header_t is not defined.

```
Traceback (most recent call last):
  File "./loxigen.py", line 176, in <module>
    lang_module.generate(options.install_dir)
  File "/home/bsn/workspace/Organization_loxigen_PR-636-IVHAZVYHIZ5OV4QL2OQZ7AR7AOERX5COOOHWIZVMSQFMDJ7WHSOQ/lang_c.py", line 125, in generate
    fn(outfile, os.path.basename(name))
  File "/home/bsn/workspace/Organization_loxigen_PR-636-IVHAZVYHIZ5OV4QL2OQZ7AR7AOERX5COOOHWIZVMSQFMDJ7WHSOQ/c_gen/c_test_gen.py", line 1796, in dup_c_gen
    gen_version_dup(out)
  File "/home/bsn/workspace/Organization_loxigen_PR-636-IVHAZVYHIZ5OV4QL2OQZ7AR7AOERX5COOOHWIZVMSQFMDJ7WHSOQ/c_gen/c_test_gen.py", line 1739, in gen_version_dup
    gen_dup_cls(out, cls, version)
  File "/home/bsn/workspace/Organization_loxigen_PR-636-IVHAZVYHIZ5OV4QL2OQZ7AR7AOERX5COOOHWIZVMSQFMDJ7WHSOQ/c_gen/c_test_gen.py", line 1657, in gen_dup_cls
    """  % dict(m_type=m_type, v_name=var_name_map(m_type)))
  File "/home/bsn/workspace/Organization_loxigen_PR-636-IVHAZVYHIZ5OV4QL2OQZ7AR7AOERX5COOOHWIZVMSQFMDJ7WHSOQ/c_gen/c_test_gen.py", line 121, in var_name_map
    return _var_name_map[m_type]
KeyError: 'of_header_t'
```

Reply:
(you should see the same result if you run `make` on your local code).